specification.markdown

```diff
@@ -386,41 +386,60 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLParentPath] EBMLElementPath
-EBMLParentPath        = EBMLFixedParent EBMLLastParent
-EBMLFixedParent       = *(EBMLPathAtom)
-EBMLElementPath       = EBMLPathAtom / EBMLPathAtomRecursive
-EBMLPathAtom          = PathDelimiter EBMLAtomName
-EBMLPathAtomRecursive = "(1*(" EBMLPathAtom "))"
-EBMLLastParent        = EBMLPathAtom / EBMLGlobalParent
-EBMLGlobalParent      = "(" GlobalParentOccurence "\)"
-EBMLAtomName          = 1*(EBMLNameChar)
-EBMLNameChar          = ALPHA / DIGIT / "-" / "."
+EBMLFullPath          = EBMLParentPath EBMLElement
+
+EBMLParentPath        = PathDelimiter [EBMLParents]
+
+EBMLParents           = 0*IntermediatePathAtom EBMLLastParent
+IntermediatePathAtom  = EBMLPathAtom / GlobalPlaceholder
+EBMLLastParent        = EBMLPathAtom / GlobalPlaceholder
+
+EBMLPathAtom          = [IsRecursive] EBMLAtomName PathDelimiter
+EBMLElement           = [IsRecursive] EBMLAtomName
+
 PathDelimiter         = "\"
-GlobalParentOccurence = [PathMinOccurrence] "*" [PathMaxOccurrence]
+IsRecursive           = "+"
+EBMLAtomName          = ALPHA / DIGIT 0*EBMLNameChar
+EBMLNameChar          = ALPHA / DIGIT / "-" / "."
+
+GlobalPlaceholder     = "(" GlobalParentOccurence "\)"
+GlobalParentOccurence = [PathMinOccurrence] "-" [PathMaxOccurrence]
 PathMinOccurrence     = 1*DIGIT ; no upper limit
 PathMaxOccurrence     = 1*DIGIT ; no upper limit
 ```
 
 The `*`, `(` and `)` symbols are interpreted as defined in [@!RFC5234].
 
-The EBMLPathAtom part of the EBMLElementPath MUST be equal to the name attribute of the EBML Schema.
+The EBMLAtomName of the EBMLElement part MUST be equal to the `@name` attribute of the EBML Schema.
+If the EBMLElement part contains an IsRecursive part, the EBML Element can occur within itself recursively (see (#recursive)).
 
-The starting PathDelimiter of the path corresponds to the root of the EBML Document.
+The starting PathDelimiter of EBMLParentPath corresponds to the root of the EBML Document.
 
-In some cases the EBMLLastParent part of the path is an EBMLGlobalParent. A path with a EBMLGlobalParent defines a Global Element; see (#global-elements). Any path that starts with the EBMLFixedParent of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element. 
+The `@path` value MUST be unique within the EBML Schema. The `@id` value corresponding to this `@path` MUST NOT be defined for use within another EBML Element with the same EBMLParentPath as this `@path`.
 
-The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition. The repetition amounts correspond to the amount of unspecified Parent Element levels there can be between the EBMLFixedParent and the actual EBMLElementPath.
+A path with a GlobalPlaceholder as the EBMLLastParent defines a Global Element; see (#global-elements).
```

Comment:
But what about a path with a GlobalPlaceholder as an IntermediatePathAtom?

Reply:
(One of) its parent is a global element, but that element in particular is not a global element. The global "placement" only applies to that parent.